modules/querying/pages/func/mathematical-functions.adoc

```diff
@@ -500,19 +539,19 @@ A numeric type.
 | `INT`, `FLOAT` or `DOUBLE`
 
 | `decimal_place`
-| Optional. The integer indicating the decimal place to truncate the number to.   If `decimal_plac` is positive, the function returns the number truncated to `decimal_place` decimal places. If `decimal_place` is omitted, then the number is truncated to 0 places. `decimal_place` can be negative to truncate (make zero) `decimal_place` digits left of the decimal point.
+| Optional. The output's number of significant digits to the right of the decimal point.
 | `INT`
 |===
 
 
 ==== Examples
 
-[,text]
+[source,text]
 ----
-trunc(9.99) => 9
-trunc(-9.99) => 9
```

Comment:
This was a typo here. We can add another example to show how negative numbers are still negative after truncation. (`trunc(-9.9) => -9`)